cmd/cli.go

```diff
@@ -281,6 +281,20 @@ func parseConfig(args []string) (*stern.Config, error) {
 		opts.since = 172800000000000 // 48h
 	}
 
+	namespaces := []string{}
+	if opts.namespaces != nil {
+		nset := map[string]struct{}{}
+		for _, n := range opts.namespaces {
+			nset[n] = struct{}{}
+		}
+		for n := range nset {
+			if n == "" {
+				continue
+			}
+			namespaces = append(namespaces, n)
+		}
+	}
+
```

Comment:
Please add a comment that makes the array unique.

Reply:
The below may be easier to read.

```suggestion
	namespaces := []string{}
	if opts.namespaces != nil {
		m := make(map[string]struct{})
		for _, namespace := range opts.namespaces {
			if namespace == "" {
				continue
			}

			if _, ok := m[namespace]; !ok {
				m[namespace] = struct{}{}
				namespaces = append(namespaces, namespace)
			}
		}
	}


```

Reply:
106178b 656cecc
I corrected the points that were pointed out!